### PR TITLE
Retrieve countrly list from server

### PR DESCRIPTION
**Pull request by DanSallau**
Get residence list from server instead.

### CODE REVIEW

File: src/_data/LiveData.js [hunk 1]
```diff
@@ -133,6 +134,6 @@ export const connect = async store => {
         api.events.on(key, data => store.dispatch(action(data)));
         api.events.on(key, () => window.console.warn);
     });
-
+    api.getResidences();
```
Comment (qingweibinary) [inline src/_data/LiveData.js]: This is not the place man ~~ 
See where we place such calls, in the same file

Comment (nuruddeensalihu) [inline src/_data/LiveData.js]: @qingweibinary that call either belong there , or veryfEmailCard to my understanding. "See where we place such calls" like ?

Comment (qingweibinary) [inline src/_data/LiveData.js]: Sorry, we've removed the initUnAuth, my bad

I think we should call when needed

Comment (borisyankov) [inline src/_data/LiveData.js]: @nuruddeensalihu I agree with @qingweibinary 
Please do the changes before me merging this.

Comment (nuruddeensalihu) [inline src/_data/LiveData.js]: @borisyankov where do we need to change ? I think if you agree with qingwei then you agree with me by extension. 😉

Comment (qingweibinary) [inline src/_data/LiveData.js]: `we should call when needed` ;)

**Comment by borisyankov** [comment]
Also fix merge conflict ;)

**Pull request merged**